From be9e8cb35692ef0b7d835dc26b28996cf6aacb7b Mon Sep 17 00:00:00 2001
From: Patrick Griffis <tingping@tingping.se>
Date: Fri, 7 Apr 2017 14:45:51 -0400
Subject: [PATCH] Fix exception with global usage

Fixes #439
---
 pithos/util.py | 7 +++----
 1 file changed, 3 insertions(+), 4 deletions(-)

diff --git a/pithos/util.py b/pithos/util.py
index e14f787..cd8d3a4 100644
--- a/pithos/util.py
+++ b/pithos/util.py
@@ -32,6 +32,8 @@
 
 # TODO: Async
 def unlock_keyring():
+    global _current_collection
+
     service = Secret.Service.get_sync(
         Secret.ServiceFlags.NONE,
         None,
@@ -49,8 +51,6 @@ def unlock_keyring():
             'Could not get the default Secret Collection.\n'
             'Attempting to use the session Collection.'
         )
-
-        global _current_collection
         _current_collection = Secret.COLLECTION_SESSION
         return
 
@@ -63,12 +63,12 @@ def unlock_keyring():
         )
 
         if not num_items or default_collection not in unlocked:
-            global _current_collection
             _current_collection = Secret.COLLECTION_SESSION
             logging.debug('The default keyring is locked. Using session collection.')
         else:
             logging.debug('The default keyring was unlocked.')
 
+
 def get_account_password(email):
     return Secret.password_lookup_sync(_ACCOUNT_SCHEMA, {"email": email}, None) or ''
 
@@ -139,4 +139,3 @@ def open_browser(url, parent=None, timestamp=0):
     popup_at_pointer = Gtk.Menu.popup_at_pointer
 else:
     popup_at_pointer = lambda menu, event: menu.popup(None, None, None, None, event.button, event.time)
-
