diff -ur pithos-1.0.0.orig/pithos/plugins/scrobble.py pithos-1.0.0/pithos/plugins/scrobble.py
--- pithos-1.0.0.orig/pithos/plugins/scrobble.py	2014-05-01 18:33:16.000000000 -0400
+++ pithos-1.0.0/pithos/plugins/scrobble.py	2014-06-07 22:42:25.380387459 -0400
@@ -14,7 +14,7 @@
 #with this program.  If not, see <http://www.gnu.org/licenses/>.
 ### END LICENSE
 
-import pylast
+import pithos.pylast as pylast
 import logging
 from pithos.gobject_worker import GObjectWorker
 from pithos.plugin import PithosPlugin
Only in pithos-1.0.0/pithos/plugins: scrobble.py~
diff -ur pithos-1.0.0.orig/setup.py pithos-1.0.0/setup.py
--- pithos-1.0.0.orig/setup.py	2014-05-01 18:33:16.000000000 -0400
+++ pithos-1.0.0/setup.py	2014-06-07 22:40:15.051748082 -0400
@@ -74,7 +74,6 @@
         ]
     },
     install_requires=[
-        'pylast'
     ],
     packages=find_packages(),
     include_package_data=True,
Only in pithos-1.0.0: setup.py~
